pypeit/spec2dobj.py

```diff
@@ -531,28 +578,9 @@ def build_primary_hdr(self, raw_header, spectrograph, master_key_dict=None, mast
         hdr['PYP_SPEC'] = spectrograph.name
         hdr['DATE-RDX'] = str(datetime.date.today().strftime('%Y-%m-%d'))
 
-        # MasterFrame info
```

Comment:
Is this information still being passed?

Reply:
Not in exactly the same format.  The header now has the calibration directory and the name of all the relevant processed calibration files.  An example of the relevant header section looks like this:
```
CLBS_DIR= '/Users/westfall/Work/packages/PypeIt-development-suite/REDUX_OUT/ke&'
CONTINUE  'ck_mosfire/K_long/Calibrations'
CLBS_ARC= 'Arc_A_11_DET01.fits'
HIERARCH CLBS_WAVECALIB = 'WaveCalib_A_11_DET01.fits'
HIERARCH CLBS_FLAT = 'Flat_A_10-11_DET01.fits'
HIERARCH CLBS_EDGES = 'Edges_A_10-11_DET01.fits.gz'
HIERARCH CLBS_SLITS = 'Slits_A_10-11_DET01.fits.gz'
HIERARCH CLBS_TILTIMG = 'Tiltimg_A_11_DET01.fits'
HIERARCH CLBS_TILTS = 'Tilts_A_11_DET01.fits'
```